BuildAndTest.proj

```diff
@@ -17,14 +17,16 @@
     <RunTestArgs>$(RunTestArgs) -nocache</RunTestArgs>
     <IncludePattern Condition="'$(IncludePattern)' == '' AND '$(TestVsi)' != 'true'">*.UnitTests.dll</IncludePattern>
     <IncludePattern Condition="'$(IncludePattern)' == '' AND '$(TestVsi)' == 'true'">*.IntegrationTests.dll</IncludePattern>
-    <OutputDirectory>Binaries\$(Configuration)</OutputDirectory>
+    <OutputDirectory>$(MSBuildThisFileDirectory)Binaries\$(Configuration)\</OutputDirectory>
     <RunTestArgs>$(RunTestArgs) -log:&quot;$(OutputDirectory)\runtests.log&quot;</RunTestArgs>
     <CoreClrTestDirectory>$(OutputDirectory)\CoreClrTest</CoreClrTestDirectory>
     <MSBuildCommonProperties>
       RestorePackages=false;
       TreatWarningsAsErrors=true;
       DeployExtension=false;
     </MSBuildCommonProperties>
+    <TestDesktop Condition="'$(TestDesktop)' == ''">true</TestDesktop>
+    <TestCoreClr Condition="'$(TestCoreClr)' == ''">true</TestCoreClr>
```

Comment:
Yep.  We can teach them to use `Test.cmd`.  I actually want to delete this file entirely because it's just a glorified script at this point.